plugin/trino-bigquery/pom.xml

```diff
@@ -580,6 +581,7 @@
                         <configuration>
                             <includes>
                                 <include>**/TestBigQueryCaseInsensitiveMapping.java</include>
+                                <include>**/TestBigQueryCaseInsensitiveMappingWithCacheDisabled.java</include>
```

Comment:
Cache is an opt-in feature. I would rename `TestBigQueryCaseInsensitiveMapping` to `TestBigQueryCaseInsensitiveMappingWithCache` and call the new test to `TestBigQueryCaseInsensitiveMapping`.